Explorer/Assets/DCL/Navmap/FloatingPanelController.cs

```diff
@@ -193,14 +197,14 @@ private void SetupDestinationButtons(bool parcelIsDestination)
             view.removeMapPinDestinationButton.gameObject.SetActive(parcelIsDestination);
             view.removeDestinationButton.gameObject.SetActive(parcelIsDestination);
         }
-
+        
         private void JumpIn(Vector2Int parcel)
         {
             OnJumpIn?.Invoke(parcel);
 
             if (destination == parcel) { mapPathEventBus.ArrivedToDestination(); }
 
-            realmNavigator.TryInitializeTeleportToParcelAsync(parcel, cts.Token).Forget();
+            chatMessagesBus.Send($"/goto {parcel.x},{parcel.y}");

```

Comment:
Maybe instead of using "/goto" we could make public the COMMAND_GOTO from the IChatCommand (or better, create a unified static class with all these commands?)? 'coz if the command changes in the future, this breaks xD

Reply:
Sounds good! Done